components/frontend/src/widgets/DivWithHTML.js

```diff
@@ -0,0 +1,32 @@
+import { createRef, useLayoutEffect, useState } from "react"
+
+import { childrenPropType } from "../sharedPropTypes"
+
+export function DivWithHTML({ children }) {
+    const minHeight = 15
+    const maxHeight = 60
+    const ref = createRef()
+    // Keep track of whether the div is overflown, so the resize handle can be hidden when not needed
+    const [overflown, setOverflown] = useState(false)
+    const [scrollHeight, setScrollHeight] = useState(minHeight)
+    useLayoutEffect(() => {
+        setOverflown(ref.current.clientHeight < ref.current.scrollHeight)
+        setScrollHeight(ref.current.scrollHeight)
+    }, [ref])
+    return (
+        <div
+            ref={ref}
+            style={{
+                wordBreak: "break-word",
+                overflow: "auto",
+                minHeight: minHeight,
+                height: Math.min(maxHeight, scrollHeight),
+                resize: overflown ? "vertical" : "",
```

Comment:
So this means that the div can only grow vertically, i.e. in height, and will therefore not break table columns that it's in?

Reply:
Yes!